### ISSUE TITLE
Types definition for suggestions possibly out of date

### ISSUE THREAD

**Issue by kahlan88** [devbridge/jQuery-Autocomplete#785]
I can see that `suggestions` allow `AutocompleteSuggestion[]` only 
```
interface AutocompleteResponse {
    suggestions: AutocompleteSuggestion[];
}
```
The docs state that `suggestions` can also be an array of strings like so:
```
{
    "query": "Unit",
    "suggestions": ["United Arab Emirates", "United Kingdom", "United States"]
}
```

I think it should therefore be:
```
interface AutocompleteResponse {
    suggestions: AutocompleteSuggestion[] | string[];
}
```
Am I correct? Is this something that could be added? If so, I would be happy to, if someone could point me to the right place to add them (would it be in this project or somewhere in DefinitelyTyped)? It seems like types have been written for jQuery-Autocomplete `1.2.25` and I'm looking at `1.4.10`.

**Comment by tkirda** [comment]
You are correct. Typings can be updated. In terms of versioning, I think it is safe to use typings with 1.4, contract has not changed.

Feel free to update: https://github.com/devbridge/jQuery-Autocomplete/blob/master/typings/jquery-autocomplete/jquery.autocomplete.d.ts and issue PR.

**Comment by tkirda** [comment]
The project just shipped **v2.0** — a ground-up TypeScript rewrite. Items previously reported in this area are no longer reproducible against current `master`:

- jQuery deprecation warnings (`$.trim`, `$.isFunction`, etc.) — source no longer uses any of them; only native equivalents.
- Bundler / ESM imports — 2.0 ships a proper ESM bundle (`dist/jquery.autocomplete.esm.js`) with correct `package.json` `exports`.
- TypeScript typings — 2.0 emits bundled `.d.ts` files from source.
- Paste / programmatic value changes — the plugin listens for the `input` event, which fires on paste in modern browsers.
- CI tooling — now handled by GitHub Actions.

Closing as resolved by 2.0. If you can still reproduce against current `master`, please open a fresh issue with a minimal repro. Thanks for the original report.
